src/plots/plots.js

```diff
@@ -1564,15 +1570,20 @@ plots.extendObjectWithContainers = function(dest, src, containerPaths) {
             if(!srcContainer) continue;
 
             destProp = Lib.nestedProperty(dest, containerPaths[i]);
-
             destContainer = destProp.get();
+
             if(!Array.isArray(destContainer)) {
                 destContainer = [];
                 destProp.set(destContainer);
             }
 
             for(j = 0; j < srcContainer.length; j++) {
-                destContainer[j] = plots.extendObjectWithContainers(destContainer[j], srcContainer[j]);
+                var srcObj = srcContainer[j];
+
+                if(srcObj === null) destContainer[j] = {};
```

Comment:
I like this. Could be null, but seems the most robust/backwards-compatible if it's at least an object.